File: interface/src/main/java/cn/itcast/core/service/SecKillOrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.seckill.SeckillOrder;

import java.util.List;

public interface SecKillOrderService {
    public PageResult findPage(Integer page, Integer rows, SeckillOrder seckillOrder);
   public void deleteById(Long[] ids);

}

File: web_manager/src/main/webapp/js/service/seckillorderService.js
//服务层
app.service('seckillorderService',function($http){

	//删除
	this.dele=function(ids){
		return $http.get('../seckillorder/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckillorder/search.do?page='+page+"&rows="+rows,searchEntity);
	}    	
});
